### DISCUSSION TITLE
Remove existing token

### DISCUSSION THREAD

**Discussion by Muscule** [chaos-mesh/chaos-mesh#1846] [Q&A] [answered]
Hello!

If I press "-" near the token, nothing happens. User's token expired, and the user can't change it.

**Comment by AsterNighT** [comment]
Hello there, can you describe your problem more specifically? Like where did you press the "-"? A screenshot would be very helpful. And it seems that you should put this question under issues rather that discussion.

**Reply by Muscule** [reply]
![screenshot1](https://user-images.githubusercontent.com/14834949/118588062-5d239800-b7a6-11eb-97f4-73c94b8ae2dc.png)

I press red button with "-" on it. And I put this question under issues yesterday.

**Reply by Muscule** [reply]
And there is another user, which I switch to before trying to remove this one.

**Reply by AsterNighT** [reply]
Great! let us discuss there.

**Comment by g1eny0ung** [comment, marked as answer]
Solved in #1875.

**Comment by Muscule** [comment]
Is there any way for cluster admin to see all users, who use chaosmesh?

**Reply by g1eny0ung** [reply]
Sorry for the late reply. I think you can label the `ServiceAccount`s to list who can access Chaos Mesh.

**Reply by Muscule** [reply]
I meant users currently using chaosmesh.

**Reply by g1eny0ung** [reply]
Currently cant.
